devolving_music/lib/song_scores.py

```diff
@@ -0,0 +1,117 @@
+from typing import Iterable
+from bisect import bisect_left
+
+
+from devolving_music.lib.elo_scoring import elo_rating
+from devolving_music.models.event import Event
+from devolving_music.models.song_submission import SongSubmission
+from devolving_music.models.song_comparison import SongComparison
+from devolving_music.lib.score_object import ScoreObject
+
+import pdb 
+#use for debugging
+# pdb.set_trace()
+
+class SongScores():
+    def __init__(self, event: Event):
+        # self.song_submissions = SongSubmission.get_voteable_submissions(Event)
+        self.song_submissions = SongSubmission.get_voteable_submissions_dict(event)
```

Comment:
Neither the keys nor values of this are submissions; call it `self.song_scores_by_id` or similar.